sdk/storage/Swagger/Generator/package-lock.json

```diff
@@ -0,0 +1,935 @@
+{
```

Comment:
Should the package-lock files be checked in?

Reply:
My understanding is that's the best practice